doc/en/example/parametrize.rst

```diff
@@ -411,10 +411,32 @@ is to be run with different sets of arguments for its three arguments:
 Running it results in some skips if we don't have all the python interpreters installed and otherwise runs all combinations (5 interpreters times 5 interpreters times 3 objects to serialize/deserialize)::
 
    . $ pytest -rs -q multipython.py
-   ...sss...sssssssss...sss...                                          [100%]
-   ========================= short test summary info ==========================
-   SKIP [15] $REGENDOC_TMPDIR/CWD/multipython.py:29: 'python3.4' not found
-   12 passed, 15 skipped in 0.12 seconds
+   Traceback (most recent call last):
```

Comment:
It seems something went wrong with regendocs... is this the error you are having @RonnyPfannschmidt?